docker/vars/Debian.yml

```diff
@@ -0,0 +1,18 @@
+---
+
+docker_package_dependencies:
+  - apt-transport-https
+  - ca-certificates
+  - curl
+  - gnupg2
+  - software-properties-common
+
+docker_apt_key: 'https://download.docker.com/linux/{{ ansible_distribution | lower }}/gpg'
+docker_apt_url: 'https://download.docker.com/linux/{{ ansible_distribution | lower }}'
```

Comment:
Will Ansible not complain about missing variables as `docker_rpm_*` or `docker_apt_*` respectively aren't set?

Reply:
No, because the task run only when the corresponding `ansible_os_family` is set.

Reply:
I do remember Ansible displaying warnings if variables weren't set. LGTM though :)